tools/ci_tools.py

```diff
@@ -101,7 +101,7 @@ def bump_file_versions(version):
 
     # bump pyproject.toml
     filename = "pyproject.toml"
-    regex = "version = \"(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(-((0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(\.(0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(\+([0-9a-zA-Z-]+(\.[0-9a-zA-Z-]+)*))?\" # OpenPype"
+    regex = "version = \"(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(\+((0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(\.(0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(\+([0-9a-zA-Z-]+(\.[0-9a-zA-Z-]+)*))?\" # OpenPype"
```

Comment:
just a note that in semver, you can have `+` for the build number. This regex won't match `
![image](https://user-images.githubusercontent.com/33513211/188193143-8fe07046-9339-45d9-a6c4-c3d6145aa390.png)